packages/webapp-libs/webapp-contentful/src/routes/demoItems/demoItemListItem/demoItemListItem.component.tsx

```diff
@@ -21,26 +20,45 @@ export const DemoItemListItem = ({ id, item }: DemoItemListItemProps) => {
   const generateLocalePath = useGenerateLocalePath();
 
   return (
-    <Container>
-      <Link to={generateLocalePath(RoutesConfig.demoItem, { id })}>
-        <FavoriteIcon
-          role="checkbox"
-          aria-checked={isFavorite}
-          aria-label={intl.formatMessage({
-            defaultMessage: 'Is favorite',
-            id: 'Demo Item / Is favorite',
-          })}
-          onClick={(e) => {
-            e.preventDefault();
-            setFavorite(!isFavorite);
-          }}
-        >
-          <Icon icon={isFavorite ? favoriteIconFilled : favoriteIconOutlined} />
-        </FavoriteIcon>
-
-        {item.image && <Thumbnail {...imageProps(item.image, { size: { height: 50 } })} role="presentation" />}
-        <Title>{item.title}</Title>
+    <li className="flex items-center w-[100%]">
+      <Link
+        className="p-8 w-[100%] justify-start hover:no-underline"
+        to={generateLocalePath(RoutesConfig.demoItem, { id })}
+      >
+        {item.image && (
+          <img
```

Comment:
Missing alt prop

Reply:
alt prop is provided by the `imageProps` function